pkgs/modules/python/default.nix

```diff
@@ -3,6 +3,8 @@
 let
   community-version = lib.versions.majorMinor python.version;
 
+  pylibs-dir = ".pythonlibs";
```

Comment:
is this a standard directory in pythonland?

Reply:
oh, nevermind, i see we're already using that dir

Reply:
No. I made it up.